packages/loader/container-loader/src/container.ts

```diff
@@ -995,9 +995,9 @@ export class Container
 				: combineAppAndProtocolSummary(summaryTree, this.captureProtocolSummary());
 
 		// Whether the combined summary tree has been forced on by either the supportedFeatures flag by the service or the the loader option or the monitoring context
-		const enableSummarizeProtocolTree =
-			this.mc.config.getBoolean("Fluid.Container.summarizeProtocolTree2") ??
-			options.summarizeProtocolTree;
+		const enableSummarizeProtocolTree = this.mc.config.getBoolean(
```

Comment:
So, the default value for this is now false. Is that expected? What about callers that were passing this as true?

Reply:
options.summarizeProtocolTree was used initially to test the Single-commit summary feature with the host apps. Now it is un-used option. Today, host apps even if they set `summarizeProtocolTree` as false, ODSP driver will reject 2commit summaries. So the option is misleading. The PR removed it's usage in the code.